### PR TITLE
fix crash after monorepo migration

### PR DESCRIPTION
**Pull request by ld-codes**
image

### CODE REVIEW

**Comment by ld-codes** [comment]
Further improvements:

**Internal packages**

If you're not planning to publish core and ui packages, then it makes sense to use[ Next.js internal package](https://turbo.build/repo/docs/handbook/sharing-code/internal-packages).

**Referenced tsconfig**

If you're planning to publish core and ui packages as separate npm packages, then it makes sense to use[ typescript's project references](https://www.typescriptlang.org/docs/handbook/project-references.html)

**Comment by ajthinking** [comment]
Awesome ✨ React now works! Though css needs to be included in the bundle:

## Before
image

## Now
image

File: .yarnrc.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+nodeLinker: node-modules
+
+yarnPath: .yarn/releases/yarn-3.6.1.cjs
```
Comment (ajthinking) [inline .yarnrc.yml:3]: I think this can be left out. I did not have the same yarn installed

File: package.json [hunk 2]
```diff
@@ -4,5 +4,6 @@
     "packages/core",
     "packages/ui",
     "packages/client"
-  ]
-}
\ No newline at end of file
+  ],
+  "packageManager": "yarn@3.6.1"
```
Comment (ajthinking) [inline package.json:8]: I think this can be left out. I did not have the same yarn installed

**Pull request merged**